playbooks/provisioning/openstack/scale-up.yaml

```diff
@@ -0,0 +1,68 @@
+---
+# Get current number of app nodes
+- hosts: masters[0]
+  tasks:
+  - name: Get number of app nodes
+    shell: oc get nodes -l autoscaling=app --no-headers=true | wc -l
+    register: oc_old_result
+
+# Get incrementation value
+- hosts: localhost
+  tasks:
+  - name: Set how many nodes are to be added
+    set_fact:
+      # How many nodes are to be added (1 by default)
+      increment_by: 1
+  - name: Check that the number corresponds to scaling up (not down)
+    assert:
+      that: 'increment_by | int >= 1'
+      msg: >
+        FAIL: The value of increment_by must be at least 1
+        (but it is {{ increment_by | int }}).
+  - name: Update openstack_num_nodes variable (if there was a deployment to begin with)
+    set_fact:
+      openstack_num_nodes: "{{ hostvars[groups['masters'][0]]['oc_old_result'].stdout | int + increment_by | int }}"
+    when:
+    - "'masters' in groups"
+    - hostvars[groups["masters"][0]]["oc_old_result"] is defined
+    register: result
+
+  - name: Update openstack_num_nodes variable (if there was no deployment)
+    set_fact:
+      openstack_num_nodes: "{{ increment_by }}"
+    when: result | skipped
+
+# Run provision.yaml with higher number of nodes to create a new app-node VM
+- include: provision.yaml
```

Comment:
please add a note to docs that this playbook works only w/o a bastion node (`openstack_use_bastion: False`). Otherwise, the static SSH config requires the post-provision step to be executed as a separate ansible-playbook call. For now, let's just add a note. We could fix this later for the bastion use case.